src/cards.scss

```diff
@@ -0,0 +1,139 @@
+$secondary: #616070;
+$btn-white-color: #fff;
+$btn-turquoise-color: #00acdc;
+$main-acent: #060b35;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  width: 1200px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    color: $main-acent;
+    margin-bottom: 4px;
+    margin-top: 0;
+    font-family: Roboto, sans-serif;
```

Comment:
There's no need to specify `font-family` here if it's already inherited from a parent element like `body`. This helps to keep the code DRY (Don't Repeat Yourself).